packages/dynamite/dynamite/lib/src/builder/client.dart

```diff
@@ -556,62 +555,66 @@ bool needsAuthCheck(
   return securityRequirements.isNotEmpty;
 }
 
-Iterable<String> buildAuthCheck(
+StringSink buildAuthCheck(
```

Comment:
The return value makes no sense because it is never used. You just write to the output directly.